environments/kubernetes/main.jsonnet

```diff
@@ -2,7 +2,16 @@ local app =
   (import 'kube-thanos.libsonnet') +
   (import 'telemeter.libsonnet') +
   (import 'observatorium.libsonnet') +
-  (import 'jaeger.libsonnet');
+  (import 'jaeger.libsonnet') + {
+    thanos+:: {
+      namespace:: $.observatorium.namespace,
+      externalURL:: 'https://observatorium.api.stage.openshift.com',
```

Comment:
Hm. This doesn't really seem right. Can we find something more generic or just ignore it?